shim/src/scopeHandler.js

```diff
@@ -68,8 +69,17 @@ export class ScopeHandler {
   // eslint-disable-next-line class-methods-use-this
   set(target, prop, value) {
     // Set the value on the shadow. The target itself is an empty
-    // object that is only used to prevent a froxen eval property.
+    // object that is only used to prevent a frozen eval property.
     // todo: use this.shadowTarget, for speedup
+
+    // new todo: allow modifications when target.hasOwnProperty(prop) and it
+    // is writable, assuming we've already rejected overlap (see
+    // createSafeEvaluatorFactory.factory). This TypeError gets replaced with
+    // target[prop] = value
+    if (objectHasOwnProperty(target, prop)) {
+      // todo: shim integrity: TypeError, String
+      throw new TypeError(`do not modify endowments like ${String(prop)}`);
```

Comment:
This message most likely will surface to developers, might want a more explicit explanation.

Reply:
Using ``` `${String(prop)}`  ``` necessary?

Reply:
I think we used that because `prop` might be a Symbol.. would plain `${prop}` handle those correctly?

Reply:
No, that would throw

Reply:
Amazing.

```js
`${Symbol.iterator}`
// Uncaught TypeError: Cannot convert a Symbol value to a string
```

```js
`${String(Symbol.iterator)}` 
// "Symbol(Symbol.iterator)"
```

We should post on https://wtfjs.com